osu.Game/Overlays/OverlayView.cs

```diff
@@ -42,25 +42,29 @@ private void load()
         /// <summary>
         /// Create the API request for fetching data.
         /// </summary>
-        protected abstract APIRequest<T> CreateRequest();
+        protected virtual APIRequest<T> CreateRequest() => null;
```

Comment:
this is actively a regression in code quality of this component. now it's easy for implementors of `OverlayView` to forget this has to be overridden.

i think the way to go forward is to remove `FriendDisplay`'s inheritance of `OverlayView` altogether. it does not fit the overall structure with this change at all, and all that's left after removing `CreateRequest()` and all associated logic is just an `APIStateChanged` subscription. it's so little code that it might as well be "duplicated" in that class locally.